[2023-08-22T08:58:51.139+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: csv_dag.counting_rows scheduled__2023-08-17T00:00:00+00:00 [queued]>
[2023-08-22T08:58:51.155+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: csv_dag.counting_rows scheduled__2023-08-17T00:00:00+00:00 [queued]>
[2023-08-22T08:58:51.155+0000] {taskinstance.py:1308} INFO - Starting attempt 2 of 2
[2023-08-22T08:58:51.189+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): counting_rows> on 2023-08-17 00:00:00+00:00
[2023-08-22T08:58:51.198+0000] {standard_task_runner.py:57} INFO - Started process 86029 to run task
[2023-08-22T08:58:51.213+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'csv_dag', 'counting_rows', 'scheduled__2023-08-17T00:00:00+00:00', '--job-id', '54', '--raw', '--subdir', 'DAGS_FOLDER/csv_dag.py', '--cfg-path', '/tmp/tmprlszpx41']
[2023-08-22T08:58:51.218+0000] {standard_task_runner.py:85} INFO - Job 54: Subtask counting_rows
[2023-08-22T08:58:51.320+0000] {task_command.py:410} INFO - Running <TaskInstance: csv_dag.counting_rows scheduled__2023-08-17T00:00:00+00:00 [running]> on host 908f67153615
[2023-08-22T08:58:51.479+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@my_first_dag.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='csv_dag' AIRFLOW_CTX_TASK_ID='counting_rows' AIRFLOW_CTX_EXECUTION_DATE='2023-08-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-17T00:00:00+00:00'
[2023-08-22T08:58:51.482+0000] {logging_mixin.py:150} INFO - An error occurred: [Errno 2] No such file or directory: 'downloaded.csv'
[2023-08-22T08:58:51.482+0000] {python.py:183} INFO - Done. Returned value was: -1
[2023-08-22T08:58:51.518+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=csv_dag, task_id=counting_rows, execution_date=20230817T000000, start_date=20230822T085851, end_date=20230822T085851
[2023-08-22T08:58:51.540+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-08-22T08:58:51.562+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
